Linux/debugger.cpp

```diff
@@ -1032,7 +1032,15 @@ int Debugger::GetLoadedModulesT(std::set<LoadedModule> &modules, bool set_breakp
   }
 
   if(set_breakpoint) {
-    AddBreakpoint((void *)(uint64_t)debug.r_brk, BREAKPOINT_NOTIFICATION); 
+    uint32_t inst = 0;
```

Comment:
This should work, but note that the extra steps should only be performed on x86/x64 CPUs. On ARM64 (`#ifdef ARM64`) the old behavior should be kept.

Reply:
I've just add new commit. Can you check it please? :)